### PR TITLE
build: Moves dependabot to develop branch

### LINKED ISSUE #506: Move Dependabot default branch to develop

### PR DESCRIPTION
**Pull request by cbaudouinjr**
closes #506

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    target-branch: "develop"
+    schedule:
+      interval: "daily"
+    commit-message:
```
Comment (peterkos) [inline .github/dependabot.yml:8]: on PR?

**Review by peterkos — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (peterkos) [reply .github/dependabot.yml:8]: @cbaudouinjr bump

Comment (cbaudouinjr) [reply .github/dependabot.yml:8]: This isn't done on PRs, rather it's a job that executes daily on the develop branch.

**Review by peterkos — APPROVED** [review]
LGTM

**Pull request merged**